### ISSUE TITLE
[Bug Report] Inconsistent v-icon widths

### ISSUE THREAD

**Issue by richdunajewski** [vuetifyjs/vuetify#7289]
### Versions and Environment
**Vuetify:** 1.5.14
**Vue:** 2.6.10
**Browsers:** Chrome 74.0.3729.169
**OS:** Windows 10

### Steps to reproduce
Create a v-icon element using Google's Material Design icons (the default), and use mood_good as the icon. Compare the overall element width to a v-icon created using mood_bad or mood, for example.

### Expected Behavior
Icon element should match the width of the icon graphic.

### Actual Behavior
Icon element width is quite large, larger than the actual icon graphic. This causes issues when trying to get icons to line up next to each other in a row (as in the Codepen) or trying to center the affected icon(s)  in a table cell, for example, as the icon is set to the left and the `<i>` container is much larger than the icon itself.

### Reproduction Link
<a href="https://codepen.io/richdunajewski/pen/BerVRG" target="_blank">https://codepen.io/richdunajewski/pen/BerVRG</a>

### Other comments
Seems like the width roughly correlates to the width of the icon ligature name, at the given font size (before it is hidden).

**Comment by KaelWD** [comment]
Duplicate of #6781

`mood_good` doesn't exist.

**Comment by richdunajewski** [comment]
You're totally right, sorry about that. The part that didn't make sense to me was the icon itself worked, so I thought I had it correct. Interesting that the first part renders if it's available, but I guess that makes total sense with ligatures!

**Comment by animald** [comment]
Regardless of whether the name of the icon was correct, this issue still exists. v-icon is the width of the text you enter to get the icon, if you have two icons, one with a longer name, they occupy differing spaces.

**Comment by jacekkarczmarczyk** [comment]
We kindly ask users to not comment on closed/resolved issues. If you believe that this issue has not been correctly resolved, please create a new [issue](https://issues.vuetifyjs.com) showing the regression or reach out to us in our [Discord community](https://community.vuetifyjs.com).

Thank you for your contribution and interest in improving Vuetify.

**Comment by davidkhess** [comment]
> Regardless of whether the name of the icon was correct, this issue still exists. v-icon is the width of the text you enter to get the icon, if you have two icons, one with a longer name, they occupy differing spaces.

Just for other folks experiencing the same issue that stumble onto this closed issue, what I found is that you can get his effect because you accidentally match a ligature for a Material Design icon when you were actually aiming for a Material Design Icons (MDI) icon. The former are rendered by translating the name into the glyph via font ligatures. The latter are rendered by css content rules substituting in a Unicode character.

When I get this width issue but the icon seemingly renders, it's because part of the name I wrote matches a Material Design ligature and sort of looks like the icon I was trying to get but the rest of the text of the name is still in the element causing it to grow wider (but the text is not rendered because normal characters are not represented in the font).

For instance, I did this once: mdi_lock_open_minus_outline

It rendered "lock_open" in Material Design to give me an open lock which sort of looked correct and then invisible text "mdi_" was added to the front and "_minus_outline" added to the end. Changing it to the correct "mdi-lock-open-minus-outline" fixes it.

Bottom line, whenever you see an icon creating width issues, it's likely you've got the icon name wrong and have invisible text adding space as a result.
